mythril/laser/ethereum/state/memory.py

```diff
@@ -0,0 +1,102 @@
+from typing import Union
+from z3 import BitVecRef, Extract, BitVecVal, If, BoolRef, Z3Exception, simplify, Concat
+
+from mythril.laser.ethereum import util
+
+
+class Memory:
+    def __init__(self):
+        self._memory = []
+
+    def __len__(self):
+        return len(self._memory)
+
+    def extend(self, size):
+        self._memory.extend(bytearray(size))
+
+    def get_word_at(self, index: int) -> Union[int, BitVecRef]:
+        """
+        Access a word from a specified memory index
+        :param index: integer representing the index to access
+        :return: 32 byte word at the specified index
+        """
+        try:
+            return util.concrete_int_from_bytes(
+                bytes([util.get_concrete_int(b) for b in self[index : index + 32]]), 0
+            )
+        except TypeError:
+            return simplify(
+                Concat(
```

Comment:
I recently realized that `Concat()` increases the bitwidth and operations like UDiv need same bitwidth of both lhs and rhs, so can you change the bitwidth in here?

Reply:
:sweat_smile: I had a 256 instead of an 8 here. It should be good now, and if not, the assert will catch it.